### PR TITLE
UI changed

### PR DESCRIPTION
**Pull request by AnkitaKhurana**
**Fixes issue:** #11 

**Changes:**
- Columns added to UI
- Footer Issues fixed

image

### CODE REVIEW

File: code/js/popup.js [hunk 1]
```diff
@@ -59,28 +61,51 @@ function dumpBookmarks(query)
...
-					    $('#bookmarks').append("<ul>"+"<p><strong>"+(key.replace(/\//g, ' / ')).replace(/\_/g, ' ')+"</strong></p>");
 					    for (var dd in obj[key])
 					    {
-					    	if(!((String(obj[key][dd]).toLowerCase()).indexOf("README.md".toLowerCase()) != -1))
-					    		$('#bookmarks').append("<a target='_blank' href='/code/"+key+"/"+obj[key][dd]+"'>"+"<li>"+obj[key][dd]+"</li></a>");
+					    	if(((String(obj[key][dd]).toLowerCase()).indexOf("README.md".toLowerCase()) != -1)){}
+
+					    	else{
+					    	
+					    		// if(s_no-1===Math.floor(obj[key].length/2)&&(s_no!=1))
```
Comment (AdiChat) [inline code/js/popup.js]: You must remove commented code though you can keep comments.

File: code/js/popup.js [hunk 2]
```diff
@@ -59,28 +61,51 @@ function dumpBookmarks(query)
...
+					str = str.replace(/\w\S*/g, function(txt) {
+						return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
+					});
+				   
 				    if(obj[key].length ==1 && ((String(obj[key]).toLowerCase()).indexOf("README.md".toLowerCase()) != -1))
 				    	continue;
 				    else
-				    {
+				    {					   
+					    let s_no = 1;
```
Comment (AdiChat) [inline code/js/popup.js]: This variable name (`s_no`) should be more explanatory such as `sub_result_number`.

File: code/js/popup.js [hunk 3]
```diff
@@ -45,7 +47,7 @@ function dumpBookmarks(query)
 
 	for(var pos in single_query)
 	{
-
+		// console.log(s)
```
Comment (AdiChat) [inline code/js/popup.js]: You may remove this commented code. 👍

**Comment by AdiChat** [comment]
Great work. 👏 

There are a few minor changes that you may make as per the comments above. 

On searching `linear`, there are two results and both are lined up in the same column. As our searching algorithm improves, it is expected that search results are narrowed further and on that case, having two results on the same column may seem odd. You may take a look into this and let us know if it can be resolved. 👍

**Comment by AnkitaKhurana** [comment]
@AdiChat Thank you for the suggestions, 

- I have made the required changes by deleting the comments and renaming the variables.
- Regarding the two result problem(eg. _'linear'_) I have Implemented the left to right addition of elements approach instead of Bootstrap's Top to bottom approach by using a lightweight library BrickLayer
Link of the same: https://github.com/ademilter/bricklayer

- Here is a Screenshot of the UI with _'linear'_ query
image

**Comment by AnkitaKhurana** [comment]
Update: Fixes #23

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**